gh.html

```diff
@@ -44,46 +44,63 @@
 	uP = false;
 	}	
 }
-
-function drawNeck()
```

Comment:
Pointlessly drawn out

Reply:
If you do a PR for the functions to draw the shapes, I will commit that. I think using the loop to create the neck will not be as intuitive to newbie programmers. For a beginner class, I always err on the side of most readable, even though it may be more verbose than some implementations.